rules/50-pandectas.rules

```diff
@@ -0,0 +1,2 @@
+KERNEL=="hiddev*", SUBSYSTEM=="usbmisc", ATTRS{idVendor}=="13ff", ATTRS{idProduct}=="0008", GROUP="adm", MODE=0664"
+KERNEL=="uinput", MODE="0660", GROUP="adm", OPTIONS+="static_node=uinput"
```

Comment:
Nasty, but by default flatpak does not give you full /dev access, so it not as open as you are describing.